infrastructure/helm/quantumserverless/values.yaml

```diff
@@ -221,4 +237,25 @@ kuberay-apiserver:
 # ===================
 
 keycloakEnable: true
+keycloakClientSecret: CLIENTSECRET-CHANGEME
+keycloakUserID: user
+keycloakUserPassword: passw0rd
 
+keycloak:
+  logging:
+    level: DEBUG
+  service:
+    type: NodePort
+    nodePorts:
+      http: 31059
+  auth:
+    adminUser: admin
+    adminPassword: passw0rd
```

Comment:
Not urgent but as we are doing with the user I would add a note for the admin password in the README too.